assets/js/components/notifications/UserInputSettings.js

```diff
@@ -26,23 +26,30 @@ import PropTypes from 'prop-types';
  */
 import { useInstanceId as useInstanceID } from '@wordpress/compose';
 import { __ } from '@wordpress/i18n';
+import { useEffect } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import Notification from '../legacy-notifications/notification';
-import { getTimeInSeconds } from '../../util';
+import { getTimeInSeconds, trackEvent } from '../../util';
 import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
 import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
 import PersonSittingSVG from '../../../svg/person-sitting.svg';
 const { useSelect } = Data;
 
-export default function UserInputSettings( { onCTAClick, isDimissable } ) {
+export default function UserInputSettings( { onCTAClick, onDismiss, isDismissable } ) {
 	const instanceID = useInstanceID( UserInputSettings );
 	const ctaLink = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-user-input' ) );
 	const userInputState = useSelect( ( select ) => select( CORE_USER ).getUserInputState() );
 
+	useEffect( () => {
+		if ( isDismissable ) {
+			trackEvent( 'user_input', 'prompt_notification_view' );
+		}
+	}, [ isDismissable ] );
```

Comment:
This feels somewhat inappropriate here since the other tracking events for the notification are handled outside of this component. Having it here couples the notification-specific tracking with this lower-level component.

I suggest we move this to where the other `prompt_notification*` events are sent.

Reply:
Sounds good. I've moved it to the new `UserInputPromptNotification` component.